tutorials/reprojection.ipynb

```diff
@@ -937,13 +986,15 @@
    "id": "e16c4fbe",
    "metadata": {},
    "source": [
-    "#### WGS84 projection - lat/lon (equirectangular) projection (EPSG:4326)\n",
-    "This very common one displays meridians and parallels as equally spaced vertical and horizontal lines. Conversion between x, y coordinates on the map and earth locations are very simple, and this is a very popular one for general use. However, the significant distortions inherent to this method make it impractical for many scientific application. Let's see how our data looks in this crs by reprojecting it with [rasterio.warp.reproject](https://rasterio.readthedocs.io/en/latest/api/rasterio.warp.html#rasterio.warp.reproject) with following steps:\n",
+    "#### WGS 84 projection - lat/lon (equirectangular) projection (EPSG:4326) with rasterio's `reproject()`\n",
```

Comment:
I think we should write the subsection name as "Reprojection with rasterio's `reproject()`" since the specific projection names are not mentioned in the title of other sections above.